doc/introduction/compiling_circuits.rst

```diff
@@ -254,4 +254,48 @@ observables and coefficients:
 >>> coeffs_groupings
 [[0.97, 4.21], [1.43]]
 
-This and more logic to manipulate Pauli observables is found in the :mod:`~.pennylane.grouping` module.
\ No newline at end of file
+This and more logic to manipulate Pauli observables is found in the :mod:`~.pennylane.grouping` module.
+
+Simplifying Operators
+----------------------
+
+PennyLane offers the :func:`~.pennylane.simplify` function to simplify single operators, quantum
+functions, QNodes and tapes. This function reduces the arithemtic depth of the given operators
+to its minimum, groups like terms in sums and products, resolves products of Pauli 
+operators and combines identical rotation gates by summing its angles.
+
+For example, lets simplify the following operator:
+
+>>> nested_op = qml.prod(qml.prod(qml.PauliX(0), qml.op_sum(qml.RX(1, 0), qml.PauliX(0))), qml.RX(1, 0))
+>>> qml.simplify(nested_op)
+PauliX(wires=[0]) @ RX(2.0, wires=[0]) + RX(1.0, wires=[0])
+
+Several simplifications steps are happening here. First of all, the nested products are removed:
+`qml.prod(qml.PauliX(0), qml.op_sum(qml.RX(1, 0), qml.PauliX(0)), qml.RX(1, 0))`
```

Comment:
Sphinx needs double back-ticks for code 
```suggestion
``qml.prod(qml.PauliX(0), qml.op_sum(qml.RX(1, 0), qml.PauliX(0)), qml.RX(1, 0))``
```

Though a code block of that length probably shouldn't be inside a paragraph.